backend/src/handlers/question_option.ts

```diff
@@ -57,8 +55,91 @@ const update = async (event: APIGatewayEvent): Promise<ProxyResult> => {
 	}
 };
 
-// DELETE /api/v1/question_option
-const remove = async (event: APIGatewayEvent): Promise<ProxyResult> => {
+export const updateMany: APIGatewayProxyHandler = async (event) => {
+	const questionId: number = Number.parseInt(
+		event.pathParameters?.questionId as string,
+		10
+	);
+	let body: QuestionlOptionAttributes[];
+
+	if (!questionId || Number.isNaN(questionId)) {
+		return responses.badRequest({
+			message: 'Question id was not set',
+		});
+	}
+
+	// lets check if they body was propperly formatted
+	try {
+		body = JSON.parse(event.body || '[]');
+	} catch (error) {
+		return responses.badRequest({
+			message: 'Body was not proper JSON',
+		});
+	}
+
+	if (!Array.isArray(body)) {
+		return responses.badRequest({
+			message: 'Missing body or what not an array of question options',
```

Comment:
I think this might be a typo: `body or what not an array`

Reply:
What should I make it say?

Reply:
`Missing body or body is not an array of question options` or `Missing body or not an array of question options`. It just seems like that `what` is a typo is all.

Reply:
nvm tell me if the new version makes more sense

Reply:
Yeah it makes sense 👍